db/migrate/20220324164502_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.text :title
```

Comment:
- [ ] For the `title` column you have to keep the data type as `string`. the previous change was only for `text` column.

Hint: 

```ruby
create_table :posts do |t|
    t.string :title
    t.text :text
    ...
  end
```